color/src/color.rs

```diff
@@ -107,6 +107,42 @@ pub enum HueDirection {
     // NOTICE: If a new value is added, be sure to modify `MAX_VALUE` in the bytemuck impl.
 }
 
+/// Defines how to handle alpha during interpolation.
+#[derive(Clone, Copy, Default, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[repr(u8)]
+#[expect(
+    clippy::exhaustive_enums,
+    reason = "There are only two ways to do interpolation."
+)]
+pub enum AlphaInterpolationSpace {
```

Comment:
Should we name this `AlphaType` so that we can use it in other places too?

Reply:
I'm thinking that we may well want to use this in the `Image` type (along with the existing `ImageFormat` specifier).

Reply:
This is a good reason for the change of the original name of `AlphaInterpolationSpace` to `AlphaMode`, but on the other hand, it feels as if it's now slightly conflating two things: namely how color channels are stored (which should not impact end results), and how color channels are treated during interpolation (which definitely results in observable behavior). It may not matter too much, though.

Reply:
Yes, good point.

Reply:
I will move back to `AlphaInterpolationSpace`